README.md

```diff
@@ -34,7 +34,12 @@ $ pip install .
 If you plan to develop the package, you may run it directly from source:
 
 ```
-$ python setup.py develop       # Do this once to add pkg to Python path
+$ pip install -e .
+```
+
+Each time after modifying a Cython or C file, also run:
```

Comment:
@stefanv unless I'm mistaken, `pip install -e .` will rebuild as well, so this could be simplified to, "Re-run this command each time you modify a Cython or C file".